destination/config.go

```diff
@@ -82,3 +91,32 @@ func (c Config) GetIndex() string {
 func (c Config) GetType() string {
 	return c.Type
 }
+
+// IndexFunction returns a function that determines the index for each record individually.
+// The function might be returning a static index name.
+// If the index is neither static nor a template, an error is returned.
+func (c Config) IndexFunction() (f IndexFn, err error) {
+	// Not a template, i.e. it's a static index name
+	if !strings.HasPrefix(c.Index, "{{") && !strings.HasSuffix(c.Index, "}}") {
```

Comment:
Now that I think about it, we should probably use `strings.Contains()` to check for the presence of `{{` and `}}`. The reasoning is that someone might want to use a static value before/after the template, for example `conduit_{{ index .Metadata "opencdc.collection" }}` would be a valid Go tamplate.

I'll make sure to open an issue for this in the Postgres connector, where this code was taken from.

Reply:
Done.